Raven.Database/Server/RavenDBOptions.cs

```diff
@@ -36,6 +36,7 @@ public RavenDBOptions(InMemoryRavenConfiguration configuration, DocumentDatabase
             
             try
             {
+                ThreadPool.SetMinThreads(configuration.MinThreadPoolWorkerThreads, configuration.MinThreadPoolCompletionThreads);
```

Comment:
This forces us by default to use 2 threads, but the actual default values are higher, IIRC.
We need to make sure that we only call this if the value is explicitly set by the user. I think that we can do that with nullable int values.